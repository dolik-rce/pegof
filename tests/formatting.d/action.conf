input formatting.d/action.peg
