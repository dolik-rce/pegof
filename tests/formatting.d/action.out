X <- A B C

A <- { action1(); }

B <- "B" { action2; }

C <- "C" { action3 = "abcd"; } "\n"
